app-backend/db/src/main/scala/com/azavea/rf/database/OrganizationDao.scala

```diff
@@ -206,4 +215,36 @@ object OrganizationDao extends Dao[Organization] with LazyLogging {
       case None => throw new IllegalArgumentException(s"User not in database: ${subjectId}")
     }
   }
+
+  def addLogo(logoBase64: Organization.LogoBase64, orgID: UUID): ConnectionIO[Organization] = {
+    val bucketName = Properties.envOrElse("DATA_BUCKET", "rasterfoundry-staging-data-us-east-1")
```

Comment:
This is already available in the application configuration (I forgot) --

if you `import com.azavea.rf.common.Config`, you can extend the `Config` trait and have `dataBucket` available from there. See, e.g., project routes